jsapp/js/actions.es6

```diff
@@ -400,31 +400,31 @@ actions.resources.deployAsset.failed.listen(function(data, dialog_or_alert){
     // setContent() removes the input box, but the value is retained
     var msg;
     if (data.status == 500 && data.responseJSON && data.responseJSON.error) {
-      msg = `<code><pre>${data.responseJSON.error}</pre></code>`;
+      msg = `<pre>${data.responseJSON.error}</pre>`;
     } else if (data.status == 500 && data.responseText) {
-      msg = `<code><pre>${data.responseText}</pre></code>`;
+      msg = `<pre>${data.responseText}</pre>`;
     } else {
-      msg = t('please check your connection and try again.');
+      msg = t('Please check your connection and try again.');
```

Comment:
Minor changes like these should go into the [`msgstr` value of the djangojs.po file](https://github.com/kobotoolbox/form-builder-translations/blob/master/en/LC_MESSAGES/djangojs.po#L40). The idea behind having lower cased strings is to keep the coded strong as simple as possible and make spelling and grammar changes to the expanded `msgstr` values. (This is as an alternative to using IDs as the msgids, like `please_check_connection`.)

We do have a mix between capitalized and lower cased strings. But if we want to go through and change them all then it would be a bit of an ordeal with transifex and the existing translations. So, I'd suggest when they are one way, we keep it that way.